README.md

```diff
@@ -37,6 +37,7 @@ Read the [Roadmap](./ROADMAP.md).
 
 - __Pre-requirement__  
 ```
+golang version 1.5 or later
```

Comment:
@zhangcoldwater 
code block should contain shell command,  not a description sentence.  
maybe you can put it out.